src/components/navigation/z-app-header/styles.css

```diff
@@ -0,0 +1,99 @@
+:host {
```

Comment:
That variables define the typographic scale of the component as per design. If some product need to change the font-size (for example if it has a very long name and don't want it to wrap), it can use the component internal variables to respect the designed scale